gateways/db/account/accounts.go

```diff
@@ -0,0 +1,123 @@
+package account_db
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/rodolfoalvesg/api-banking/api/domain/entities/accounts"
+	account "github.com/rodolfoalvesg/api-banking/api/domain/usecases/accounts"
+)
+
+var (
+	ErrIDExists = errors.New("ID already exists!")
+	ErrNotFound = errors.New("Account not found")
```

Comment:
```suggestion
	ErrNotFound = errors.New("account not found")
```
Geralmente os linters vão implicar aqui, dizendo que as mensagens dos erros devem iniciar com letra minúscula.

Reply:
corrigido